### PR TITLE
Update notification

### LINKED ISSUE #392: Notification when update available
Users don't know about software updates when they are made available.

**Checking for updates**
- make request to check for software releases when client first loads

**Notification icon**
- show notification icon in navbar indicating update is available
- could replace GitHub icon link since it is pretty useless
- have some sort of pulsing animation (see transform and transition CSS properties)

**Potential for future notifications**
- show popover when hovering the notification icon
- could be expanded for notifications other than software update

### PR DESCRIPTION
**Pull request by Miminko**
- resolves #392

### CODE REVIEW

File: client/src/js/nav/components/Icon.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    componentWillMount () {
+        this.props.onGet();
+        this.props.onGetUnbuilt();
+    }
+
+    componentDidMount () {
+        this.interval = window.setInterval(() => {
+            this.props.onGet();
+            this.props.onGetUnbuilt();
+        }, 10000);
```
Comment (igboyes) [inline client/src/js/nav/components/Icon.js]: Change interval to every few hours or so. Every ten seconds is a lot of requests to _www.virtool.ca_

File: client/src/js/nav/components/Icon.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+        });
+    }
+
+    handleExit = (e) => {
+        if (this.target.contains(e.target)) {
+            return;
+        }
+
+        this.handleToggle();
+    }
```
Comment (igboyes) [inline client/src/js/nav/components/Icon.js]:
```javascript
if (!this.target.contains(e.target)) {
    this.handleToggle();
}
```

**Comment by Miminko** [comment]
The popover toggle seems to have a lag when closing; I'm not sure how to optimize this to be faster

**Comment by igboyes** [comment]
I'll check it out.

**Pull request merged**